SalesforceSDKCore.podspec

```diff
@@ -8,14 +8,12 @@ Pod::Spec.new do |s|
   s.license      = { :type => "Salesforce.com Mobile SDK License", :file => "LICENSE.md" }
   s.author       = { "Kevin Hawkins" => "khawkins@salesforce.com" }
 
-  s.platform     = :ios, "16.0"
+  s.platforms    =  { :ios => "16.0", :visionos => "2.0" }
   s.swift_versions = ['5.0']
 
   s.source       = { :git => "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git",
                      :tag => "v#{s.version}" }
 
-  s.frameworks   = 'CoreTelephony'
```

Comment:
Realized we had a CoreTelephony import in SalesforceSDKCore but it wasn't being used in the importing class